SEBrowser/Controllers/OpenXDAController.cs

```diff
@@ -219,32 +225,47 @@ private string getEventTypeFilter(EventSearchPostData postData)
 
         private string getPhaseFilter(EventSearchPostData postData)
         {
-            List<string> phase = new List<string>();
-
-            if (postData.PhaseA && postData.PhaseB && postData.PhaseC)
-                return "";
-            if (!postData.PhaseA && !postData.PhaseB && !postData.PhaseC)
+            Dictionary<string, bool> phases = new Dictionary<string, bool> 
+            {
+                ["AN"] = postData.PhaseAN,
+                ["BN"] = postData.PhaseBN,
+                ["CN"] = postData.PhaseCN,
+                ["AB"] = postData.PhaseAB,
+                ["BC"] = postData.PhaseBC,
+                ["ABG"] = postData.PhaseABG,
+                ["BCG"] = postData.PhaseBCG,
+                ["ABC"] = postData.PhaseABC,
+                ["ABCG"] = postData.PhaseABCG
+            };
+
+            bool allTrue = true;
+            bool allFalse = true;
+            foreach (var value in phases.Values) 
+            {
+                if (value)
+                {
+                    allFalse = false;
+                }
+                else
+                {
+                    allTrue = false;
+                }
+            }
+            if (allTrue || allFalse)
+            {
                 return "";
```

Comment:
This is a bug. When no Phase is selected (all False) we need to have an impossible condition to make sure we don't return any results (like `1=0`)